docs/feature-flagging/randomization-sdk/api-keys.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 1
+---
+
+# API Keys
+
+You will need to initialize the SDK with an API key. The SDK uses this key to authenticate requests to Eppo's API for retrieving your experiment variations. Do not share your API token with anyone outside your organization.
```

Comment:
nit:

I would suggest starting the sentence with `You will need to initialize whichever SDK you chose`